src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -80,11 +84,10 @@ class AdaptiveCard extends Component<IAdaptiveCardProps> {
     if (body && !pending) {
       if (!data || (queryStatus && !queryStatus.ok)) {
         return (
-          <Label className={classes.emptyStateLabel}>
+          <Label className={classes.emptyStateLabel} styles={{root: this.textStyle}}>
```

Comment:
The className isn't needed here since the styling produces the desired effect. 
You can remove it

Reply:
Done :)